src/combineReducers.js

```diff
@@ -134,6 +134,7 @@ export default function combineReducers(reducers) {
       nextState[key] = nextStateForKey
       hasChanged = hasChanged || nextStateForKey !== previousStateForKey
     }
+    hasChanged = hasChanged || Object.keys(state).length !== finalReducerKeys.length
```

Comment:
Hey, so I have been following this issue and am trying to wrap my head around the fastest way to do it.  Do you say it is too costly because calling `Object.keys` will take O(n) time, or is it a memory issue? Forgive me, I have never contributed to open source before but I want to take a stab at it.